WD-Firmware/rtos/rtosal/rtosal_event.c

```diff
@@ -29,6 +29,7 @@
 #include "rtosal_macro.h"
 #include "rtosal.h"
 #include "psp_api.h"
+#include "psp_defines.h"
```

Comment:
we should have one include for psp, it will be good for modulation (cores) and for user experience